openedx/openedx-platform
# feat: Add SSO History for Support

This PR caters to the SSO History with regard to a particular user. The primary target is to add an additional field in the Program Enrollment Inspector that shows a history of SSO links belonging to the user’s username. 

Ticket Link: [PROD-2423](https://openedx.atlassian.net/browse/PROD-2423)

Testing Instructions: 
1. Make migrations inside lms/cms:
`./manage.py lms migrate`

2. Make UserSocialAuth at `/admin/social_django/usersocialauth` and make test changes after creation. 
3. Visit `/support/sso_history/<username_or_email>` to see the list of changes.
